tests/test_homo_sapiens.py

```diff
@@ -119,3 +119,4 @@ class TestGutenkunstThreePopOutOfAfrica(
 class TestSchiffelsZigzag(
         unittest.TestCase, test_models.CatalogDemographicModelTestMixin):
     model = species.get_demographic_model("Zigzag_1S14")
+    qc_model = homo_sapiens_qc.ZigZag()
```

Comment:
This needs to be changed to instead inherit from `QcdCatalogDemographicModelTestMixin`, which should enable the QC checking unit tests for the model.

Reply:
Ok, perfect! Easy fix. Thanks.